lib/views/menu.erb

```diff
@@ -1,5 +1,21 @@
 <h1>Our Menu</h1>
 
-  <% @dishes.each do | dish | %>
-  <p><%= dish.name %>, <%= dish.price %></p>
-  <% end %>
+<h2>Categories</h2>
+
+<h3>Starters</h3>
+
+   <% @starter.each do | dish | %>
+   <p><%= dish.name %>, <%= dish.price %></p>
+   <% end %>
+
+<h3>Main Courses</h3>
+
+   <% @main_course.each do | dish | %>
+   <p><%= dish.name %>, <%= dish.price %></p>
+   <% end %>
+
+<h3>Desserts</h3>
+
+   <% @dessert.each do | dish | %>
+   <p><%= dish.name %>, <%= dish.price %></p>
+   <% end %>
```

Comment:
You should indent this file like you would regular ruby code - that is, I want to see the lines with `<p>` indented two spaces so they are "inside" the `do / end` blocks.